kit/azure/pam/README.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Privileged Access Management
+summary: |
+  Privileged Access Management (PAM) refers to the implementation of security measures and best practices to control and monitor access to critical resources within cloud platforms. For cloud foundation teams, it is about safeguarding administrative roles that enable access to core infrastructure, ensuring the security, compliance, and visibility needed to oversee application teams' cloud usage.
```

Comment:
d: should add compliance statement (e.g. implement PAM for security auditors, billing readers etc)